pymc/model.py

```diff
@@ -1856,6 +1858,74 @@ def Deterministic(name, var, model=None, dims=None, auto=False):
     return var
 
 
+def _get_scaling(total_size, shape, ndim):
```

Comment:
Can we get a direct test for all of the `_get_scaling` expected behaviors?

Reply:
This function seems to also exist here: 

https://github.com/pymc-devs/pymc/blob/b71bb74e12d06daeb6ef0facdfecc369c7ae256c/pymc/distributions/logprob.py#L46

?